docs/lib/snippets/dart_api/manager.dart

```diff
@@ -332,3 +332,65 @@ Future<void> customOrdering(AppDatabase db) async {
   db.managers.todoItems.orderBy((f) => f.contentThenCreatedAt());
 }
 // #enddocregion manager_custom_ordering
+
+void _managerAnnotations(AppDatabase db) async {
+// #docregion manager_annotations
+// First create an annotation with an expression you want to use
+  final titleLengthAnnotation =
+      db.managers.todoItems.annotation((o) => o.title.length);
+
+  /// Create a copy of the manager with the annotations you want to use
+  final manager =
+      db.managers.todoItems.withAnnotations([titleLengthAnnotation]);
+
+// Then use the annotation in a filter
+// This will filter all items whose title has exactly 10 characters
```

Comment:
I think for manager users who might not be too familiar with database operations or SQL, a point we should mention is that this check happens inside the database, speeding things up by not reading rows we don't care about into Dart. (This doesn't have to be mentioned in the comments alone, can also be expanded on in the docs)

Reply:
I'll add it to the docs, that a great point!